mk/vs2015/GitInfoPropertySheet.props

```diff
@@ -5,6 +5,14 @@
     <GitCommit>unavailable</GitCommit>
     <GitBranch>unavailable</GitBranch>
   </PropertyGroup>
+  <ItemGroup>
+  <BuildMacro Include="GitCommit">
```

Comment:
I'm not sure what `BuildMacro` actually does. Could you please elaborate?

Reply:
Without that when I open the property sheet for viewing (at least in VS2017) the list of macros shows up empty:

![image](https://user-images.githubusercontent.com/21955286/63170978-92ff7f80-c008-11e9-99af-e0da12ea3f6d.png)